### PR TITLE
Adds influx logging

### PR DESCRIPTION
**Pull request by bg451**

### CODE REVIEW

File: runner.go [hunk 1]
```diff
@@ -32,13 +33,22 @@ func init() {
 		os.Exit(2)
 	}
 	log.SetOutput(f)
+
+	config := &Config{
```
Comment (rdallman) [inline runner.go]: can this be configurable via flag or something? just don't want it on by default

Comment (rdallman) [inline runner.go]: the variables for host, db, etc would be a bonus, but really if you want then `-influx` to toggle these is fine

File: influx.go [hunk 2]
```diff
@@ -0,0 +1,122 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	influxClient "github.com/GeertJohan/influxdb-client-old"
```
Comment (rdallman) [inline influx.go:8]: use 0.8 influx ? can you doc that in readme please? no dice with 0.9?

Comment (bg451) [inline influx.go:8]: last time I checked, influxdb was still using 0.8 for their hosting+grafana service. Not sure if its been upgraded since

Comment (rdallman) [inline influx.go:8]: think you have to use a nightly build to get grafana w/ 0.9. that's ok, could you just doc influx version in README for the next person who might end up trying to use this. thanks

**Comment by rdallman** [comment]
thanks!

**Pull request merged**